gitea/statefulset.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: gitea
+  namespace: gitea
+  labels:
+    app: gitea
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gitea
+  template:
+    metadata:
+      labels:
+        app: gitea
+    spec:
+      securityContext:
+        fsGroup: 1000
+      ##  runAsUser: 1000
+      ##  runAsGroup: 1000
+      containers:
+      - name: gitea
+        image: gitea/gitea:latest
```

Comment:
Fine for now, but we should probably push an image to our Docker registry and use that.

Reply:
yes can we do in 2nd step as I believe we might need a few other things